src/packages/sdk/src/__tests__/fixtures/getSchema/conventional-path-workspaces/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "yarn-workspaces",
+  "workspaces": ["packages/*"],
+  "private": true
+}
```

Comment:
I created a bunch of fixtures to match the current style of testing. I can as well come up with some utilities to dynamically write these fixtures in `tmp` and clean them up for each tests. That's what we used to do in Nexus.